src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/DeleteCollectionCommand.java

```diff
@@ -0,0 +1,23 @@
+package io.stargate.sgv2.jsonapi.api.model.command.impl;
+
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import io.stargate.sgv2.jsonapi.api.model.command.NamespaceCommand;
+import javax.validation.constraints.NotBlank;
+import org.eclipse.microprofile.openapi.annotations.enums.SchemaType;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+/**
+ * Command for deleting a collection.
+ *
+ * @param name Name of the collection
+ */
+@Schema(description = "Command that deletes a collection if one exists.")
+@JsonTypeName("deleteCollection")
+public record DeleteCollectionCommand(
+    @NotBlank
+        @Schema(
+            description = "Name of the collection",
+            implementation = Object.class,
+            type = SchemaType.OBJECT)
```

Comment:
Shouldn't this be `String`?

Reply:
Yup fixing this in a followup PR. I was copying from the create collection command, which also has it wrong.

Reply:
https://github.com/stargate/jsonapi/pull/157